src/python/platforms/android/settings.py

```diff
@@ -174,6 +174,12 @@ def is_google_device():
   return product_brand in ('google', 'generic')
 
 
+def is_automotive():
+  """Returns if we are running in Android Automotive OS, currently only for Osprey."""
+  product_name = get_product_name()
+  return product_name == 'osprey_hwasan'
```

Comment:
i don't think hwasan is the right match, as we can have devices without hwasan, so either use model (define a function) or use startswith.

Reply:
sure, let's use model.